lib/src/test/java/vmath/Vec3Test.java

```diff
@@ -138,6 +138,16 @@ void givenTwoVectors_whenCrossProductCalculated_thenResultIsCorrect() {
         assertEquals(cross.z(), -1.0f);
     }
 
+    @Test
+    void givenThreevectors_whenTripleProdpctCalculated_thenResultIsCorrect(){
```

Comment:
Capitalize "Vectors" and fix spelling of "product" in this test name. Add space before curly brace.